### ISSUE TITLE
Add customization examples

### ISSUE THREAD

**Issue by akaleeroy** [markdown-it/markdown-it-footnote#47]
Regarding #28 what would be a good example to add to the README?
I've used the following to customize footnotes for EPUB ebooks:

```js
const backrefLabel = 'back to text';

const epubRules = {
  footnote_ref: ['<a', '<a epub:type="noteref"'],
  footnote_open: ['<li', '<li epub:type="footnote"'],
  footnote_anchor: ['<a', `<a aria-label="${backrefLabel}"`],
}

Object.keys(epubRules).map(rule => {
  let defaultRender = md.renderer.rules[rule];
  md.renderer.rules[rule] = (tokens, idx, options, env, self) => {
    return defaultRender(tokens, idx, options, env, self).replace(...epubRules[rule]);
  }
})
```

Since renderers return a string this just does string replacement to add required attributes to the markup.

Reference: [Markdown-It Architecture > Renderer](https://github.com/markdown-it/markdown-it/blob/master/docs/architecture.md#renderer)
 
Seems to have worked, but I'm not sure if this is a good example to add. Which is why I opened this issue.

**Comment by rlidwka** [comment]
Yes, it is as good of an example as you can get with the current architecture.

*Personally, I'd like to have markdown AST, transform it to HTML DOM and serialize that. Then you'd be able to write a plugin to add `epub:type` attribute between these two steps. We don't have that, so extending renderer is a reasonable approach.*

**Comment by rlidwka** [comment]
added your example to README, thank you
